src/main/java/game/data/entity/EntityRegistry.java

```diff
@@ -73,16 +70,71 @@ public void addEntity(DataTypeProvider provider, Function<DataTypeProvider, Enti
             });
             
             if (ent instanceof Villager villager) {
-                WorldManager.getInstance().getVillagerManager().loadPreviousTradeAt(villager);

```

Comment:
This unfortunately breaks villager trades saving when the villager is reloaded.

Reply:
I guess this can be fixed later, I think most people aren't too concerned with villagers anyway. Maybe someone else has time to go over it and submit a PR for it.